alex_questions.md

```diff
@@ -0,0 +1,19 @@
+# direct TX
+
+- What format can we expect for the direct TX on the DA? some wrapper type would be useful to unpack and distinct from
+  random bytes
+- Should we always include direct TX although they may be duplicates to mempool TX? spike: yes, first one wins.
+- Should we fill the block space with directTX if possible or reserve space for mempool TX
+- What should we do when a sequencer was not able to add a direct-TX within the time window?
```

Comment:
A sequencer following the rules of the network should be able to add a direct tx within the time window. If there's too many direct txs to include in a time window, then everyone can see that and those extra direct txs would just be ignored. This responsibility of telling a user if their direct tx will be included or not, or has be included can be passed on to the frontends.